src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -218,6 +229,7 @@ $pf-c-dropdown--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "
 
   &.pf-m-expanded {
     --pf-c-dropdown__toggle--m-split-button--m-primary--child--BackgroundColor: var(--pf-c-dropdown__toggle--m-split-button--m-primary--child--m-expanded--BackgroundColor);
```

Comment:
Can you leave a comment here to update this var name to `--pf-c-dropdown--m-expanded__toggle--m-split-button--m-primary--child--BackgroundColor` like you've done with the secondary var?